### DISCUSSION TITLE
跪求大佬更新！

### DISCUSSION THREAD

**Discussion by lwei65** [mikumifa/cppTickerBuy#8] [提问专区]
cp30的vip和连票好像只能在cpp上抢
跪求作者更新下cpp-buy的验证码验证（~~如有~~）
Orz  跪求 ୧⍢⃝୨

**Comment by mikumifa** [comment]
起码要验证码出现一次，我才能改，也就是如果cp30是第一次出现验证码的话，我是没办法改的
